services/api/src/routes/applications.js

```diff
@@ -0,0 +1,104 @@
+const Router = require('@koa/router');
+const { validateBody } = require('../utils/middleware/validate');
+const { authenticate, fetchUser } = require('../utils/middleware/authenticate');
+const { Application, ApplicationEntry } = require('../models');
+const { kebabCase } = require('lodash');
+const { exportValidation, csvExport } = require('../utils/csv');
+const Joi = require('joi');
+
+const router = new Router();
+
+router
+  .use(authenticate({ type: 'user' }))
+  .use(fetchUser)
+  .param('application', async (id, ctx, next) => {
+    const application = await Application.findOne({ _id: id, user: ctx.state.authUser.id });
```

Comment:
it would require a extra check ... if admin then dont check for the user

Reply:
well either that or a different param

but the bigger question is: is there a place where the admin can view all the applications?
if so would require a different route as well

Reply:
There is not a place where admins can see all applications, somewhat by design in an attempt to keep it simple and not enforce any access policy => yet leave enough body that it can easy be build out.